config/locales/en.yml

```diff
@@ -450,11 +450,30 @@ en:
         signed_on: Signed on %{date}
   home:
     show:
+      about_card_1_content: RailsDevs strives to give power back to the independent developer. Instead of companies posting their jobs, developers post their profiles.
+      about_card_1_title: Empowering the developer
+      about_card_2_content: RailsDevs is an Open Startup and open source. It operates fully transparent and shares its metrics, like revenue and traffic.
+      about_card_2_title: Doing everything in public
+      about_card_3_content: The platform is open to all developers, regardless of race, color, national origin, religion, gender, age, disability, or skill level that follow the code of conduct.
+      about_card_3_title: Creating a safe, inclusive environment
```

Comment:
Let's nest these to make them a bit easier to understand.

```yaml
home:
  show:
    about_card_1:
      content:
      title:
    about_card_2:
      content:
      title:
    # etc.
```

Reply:
TIL